src/Abp.ZeroCore/Authorization/Users/UserTokenExpirationWorker.cs

```diff
@@ -35,17 +33,8 @@ public class UserTokenExpirationWorker<TTenant, TUser> : PeriodicBackgroundWorke
             _unitOfWorkManager = unitOfWorkManager;
             _tenantRepository = tenantRepository;
 
-            Timer.Period = GetTimerPeriod();
-        }
-
-        private int GetTimerPeriod()
-        {
-            if (_backgroundJobConfiguration.CleanUserTokenPeriod.HasValue)
-            {
-                return _backgroundJobConfiguration.CleanUserTokenPeriod.Value;
-            }
-
-            return IntervalInMilliseconds;
+            var period = _backgroundJobConfiguration.UserTokenExpirationPeriod ?? TimeSpan.FromHours(1);
+            Timer.Period = _backgroundJobConfiguration.CleanUserTokenPeriod ?? Convert.ToInt32(period.TotalMilliseconds);
```

Comment:
Since `CleanUserTokenPeriod` is deprecated and `UserTokenExpirationPeriod` is nullable with no default value, let's prioritise `UserTokenExpirationPeriod`.

This may be clearer about the fallback:

```c#
Timer.Period = _backgroundJobConfiguration.UserTokenExpirationPeriod?.TotalMilliseconds.To<int>()
    ?? _backgroundJobConfiguration.CleanUserTokenPeriod
    ?? TimeSpan.FromHours(1).TotalMilliseconds.To<int>();
```

Reply:
👍